src/util/validate.py

```diff
@@ -15,6 +15,20 @@
 }
 
 
+def validate_assessment_id(assessment_id):
+    """Validate that the provided assessment_id is matching the valid assessment_id format. Example: RV1234.
+
+    Args:
+        assessment_id (string): Assessment identifier to validate.
+
+    Returns:
+        match: the result of a regular expression match.
+    """
+    match = re.match(r"^RV\d{4}", assessment_id)
```

Comment:
What happens when assessments IDs are longer than 4 digits?

Reply:
Ben and I talked about this. We figure we will be using the 4 digit RV numbers for a few years, so he thought it was okay. If we end up changing the format, we would have to revise this.

Reply:
The regex was modified and included in the previous change to make the function a one-liner. 

See c6d29c5 for revisions